recipe/meta.yaml

```diff
@@ -35,7 +22,7 @@ build:
     # I have no idea what this line actually fixes (other than speeding up the build a little bit)
     # I think windows was freaking out about path lengths, and maybe this fixed it???
     # https://github.com/conda-forge/scikit-image-feedstock/pull/39
-    - "{{ PYTHON }} setup.py build_ext -j2"
+    # - "{{ PYTHON }} setup.py build_ext -j2"
```

Comment:
maybe revert this?

Reply:
Was wondering about that too, but the commit that used this earlier in the PR had the [same error]( https://ci.appveyor.com/project/conda-forge/scikit-image-feedstock/builds/21719804/job/n54ss9ylpotc8fuo#L1490 ).

Reply:
Though the error is only happening with `pip` not during the `setup.py` build. Maybe we should try not using `pip`?

Reply:
Genius!

Reply:
I really don't know how to install with setup.py without dependencies

https://stackoverflow.com/a/30496101/2321145